packages/sdk/governance/src/proposals.ts

```diff
@@ -82,6 +81,24 @@ const registryRepointArgs = (tx: ProposalTransactionJSON) => {
   }
 }
 
+const isRegistryRepointRaw = (tx: ProposalTransaction) =>
+  tx.to === REGISTRY_CONTRACT_ADDRESS && trimLeading0x(tx.input).startsWith('c5865793')
```

Comment:
Would be nice to either have a comment explaining that `web3.eth.abi.encodeFunctionSignature('setAddressFor(string,address)') = '0xc5865793'`, or possibly to pass in abiCoder to this function and do `abiCoder.encodeFunctionSignature('setAddressFor(string,address)')` instead of hardcoding it

Reply:
great idea